core/vm/contracts.go

```diff
@@ -502,3 +526,416 @@ func (c *blake2F) Run(input []byte) ([]byte, error) {
 	}
 	return output, nil
 }
+
+var (
+	errBLS12381InvalidInputLength          = errors.New("invalid input length")
+	errBLS12381InvalidFieldElement         = errors.New("must be less than modulus")
```

Comment:
This is unused

Reply:
Unused errors that you marked are actually used in failure tests as expected errors. Should I move these next to the tests or export from bls package? Also notice that these are complete set of errors we expect from the bls precomplies.